CorpusSearch/ClientApp/src/routes/BitPlayer.tsx

```diff
@@ -0,0 +1,33 @@
+/* eslint-disable @typescript-eslint/no-unsafe-argument,@typescript-eslint/no-unsafe-assignment,@typescript-eslint/no-unsafe-member-access,@typescript-eslint/no-unsafe-call */
+import "./DocumentView.css"
+import React, {Fragment, useState} from "react"
+import YouTube, {YouTubeProps} from "react-youtube"
+
+function Video({vId="rLEBp8R1_XA"}) {
+    const [ptime, setPtime] = useState(-1)
+    const [video, setVideoId] = useState(vId)
+    const onPause : YouTubeProps["onPause"] = (event) => {
+        const player = event.target
+        setPtime(player.getCurrentTime())
+    }
+    const onChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+        setVideoId(e.target.value)
+    }
+
+    return <><Fragment>
```

Comment:
`<>` is shorthand for `<Fragment>`, both aren't necessary. 

Prefer `<>` unless you need an attribute on the fragment (typically `key=`).